examples/wifi-plotting-numpy.py

```diff
@@ -0,0 +1,124 @@
+# This is an example showing how to use the ns-3 SimulationExecutionManager to
+# get from compilation to result visualization.
+
+from sem import CampaignManager, list_param_combinations
+import os
+from pathlib import Path
+import shutil
+import re
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+# Define campaign parameters
+############################
+
+script = 'wifi-multi-tos'
+# ns_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'ns-3')
+# ns_path = '/home/davide/Work/sem/examples/ns-3'
+ns_path = '/Users/davide/Work/sem/examples/ns-3'
+campaign_dir = "/tmp/sem-test/wifi-plotting-example"
+
+# Create campaign
+#################
+
+if (Path(campaign_dir).exists()):
+    shutil.rmtree(campaign_dir)
+
+campaign = CampaignManager.new(ns_path, script, campaign_dir,
+                               runner='ParallelRunner')
+
+print(campaign)
+
+# Run simulations
+#################
+
+# Complete parameter space
+##########################
+# nWifi_values = [1, 3, 5, 7]
+# distance_values = list(range(1, 50, 10))
+# simulationTime_values = [10]
+# useRts_values = ['false', 'true']
+# mcs_values = list(range(0,8))
+# channelWidth_values = ['20', '40']
+# useShortGuardInterval_values = ['false', 'true']
+
+# Partial parameter space
```

Comment:
Partial and small parameter spaces have the same setting. Is this intentional?

Reply:
No, I'll remove the partial setting!

Reply:
Thanks. Please remove all the dead codes.